modules/standard/Barrier.chpl

```diff
@@ -41,6 +41,10 @@
        writeln("Task ", tid, " is past the barrier");
      }
 
+   Note: When a barrier instance goes out of scope it is automatically deleted.
+   After it is deleted, any copies of the barrier that remain are invalid.
+   Using a copy of a barrier after it has been deleted is undefined.
```

Comment:
I'd remove this last line entirely, but if not, I'd say "causes undefined behavior" or "is likely to cause a crash" or something slightly more grammatical and hopefully more obvious.